Pages for sharing data (#1136)
ACCESS-Hive/access-hive.github.io: docs/model_evaluation/data/sharing/sharing.md

# Sharing ACCESS Data

ACCESS-NRI has a number of options to support the ACCESS community with sharing
data at NCI, roughly divided into three areas:

- ACCESS Working Group Storage
- ACCESS Community Datapool
- ACCESS Reference Datasets

These storage options are suitable for both ACCESS model output data and other
datasets, as long as the data is of value to the ACCESS community then support
may be available.
These storage options are supplied by ACCESS-NRI and support is given based on
value to the ACCESS community.

> Regular reviews of the datasets are conducted to ensure datasets demonstrate

Reviewer: Do we review the Working Group storage datasets?

Author: That probably depends on the working group. I thought I'd keep things simple - the main point I want to convey is that support is not necessarily forever.

ongoing value.

> As the scheme matures and available storage becomes limited, recommendations

Reviewer: Is this true for all 3 storage options?

I would avoid using the term "scheme" - that sounds like language we can use internally, but our users might not know what scheme we are talking about. 

Perhaps we could say something like "These storage options are relatively new. As usage increases and we have a better understanding of community needs and interests, recommendations for changes..."

for changes, including retirement or transition of datasets, will be submitted to
the [ACCESS-NRI Scientific Advisory Committee](https://www.access-nri.org.au/about/scientific-advisory-committee-sac/)
for advice.

> 

Reviewer: ```suggestion

## Which storage option should I choose?

```

Doesn't have to be this wording, but I suggest having a header like this. I'd guess that this table is going to be the thing on this page that is most viewed/useful, so let's draw attention to it.

> The following table gives a rough guidelines for these storage options:

Reviewer: ```suggestion
The following table gives rough guidelines for usage of the three storage options:
```

|                        | Working Group | Datapool   | Reference Datasets |
| ---------------------: | :-----------: | :--------: | :----------------: |
| **Timeframe Guidline** | <1 year       | 1-2 years  | >2 years           |
| **Data State**         | Any           | Stable     | Finalised          |
| **Sharing Scope**      | Working group | Gadi       | Global             |
| **Managed by**         | Working group | ACCESS-NRI | ACCESS-NRI & NCI   |
| **Metadata Overhead**  | Minimal       | Small      | Medium             |
| **NCI Catalogue**      | ✗             | ✗          | ✓                  |
| **DOI**                | ✗             | ✗          | ✓                  |
| **Public Access**      | ✗             | ✗          | ✓                  |

## Working Group Storage

Each of the ACCESS Community working groups has a storage allocation to support
the group in their research efforts.
See this page for more information on
[joining ACCESS working groups](https://www.access-nri.org.au/join-community-working-groups/).

Working group storage is ideal for data that is under active development or only
needs to be shared within the working group.

> Each working group's storage is managed by the working group themselves.

Reviewer: ```suggestion
Each working group's storage is managed by the working group members.
```

## ACCESS Community Datapool

ACCESS-NRI maintains the ACCESS Community Datapool on Gadi under the project `av17`.
The Datapool supports datasets that are of value to be shared with the community
but may not be considered ready or appropriate for publication as a Reference Dataset.
Datapool storage is ideal for stable data that needs to be shared amongst the
ACCESS community.

The Datapool is managed by the ACCESS-NRI Data team.

> ## Reference Datasets

Author: Having data added to the NCI catalogue and getting a DOI etc. would be publishing. I'll see if I can slip that word in somewhere....
